extension-manifest-v3/src/utils/api.js

```diff
@@ -38,6 +56,7 @@ export async function setCookie(name, value, durationInSec = COOKIE_DURATION) {
     name,
     value,
     expirationDate: Date.now() / 1000 + durationInSec,
+    firstPartyDomain: (await isFirstPartyIsolation()) ? DOMAIN : '',
```

Comment:
are you sure that this param can be there on all platforms and with first party isolation disabled? I remember the API throwing exceptions when first party isolation was not supported, this is way MV2 adds this param conditionally firstPartyDomain

Reply:
I can check again to be 100% sure, but from the MDN it looks like it should be fine to set `''` - this is the state when it is not supported.

Reply:
In Safari is fine, but Chrome does not like the parameter, so you were right ;)